dropwizard-migrations/src/test/java/io/dropwizard/migrations/AbstractMigrationTest.java

```diff
@@ -0,0 +1,46 @@
+package io.dropwizard.migrations;
+
+import io.dropwizard.db.DataSourceFactory;
+import net.sourceforge.argparse4j.ArgumentParsers;
+import net.sourceforge.argparse4j.inf.Subparser;
+import org.junit.Before;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+
+public class AbstractMigrationTest {
+
+    protected TestMigrationConfiguration conf;
+    protected String databaseUrl;
+    protected final ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+    private static String createTempFile() {
+        try {
+            return File.createTempFile("test-example", null).getAbsolutePath();
+        } catch (IOException e) {
+            throw new IllegalStateException(e);
+        }
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        databaseUrl = "jdbc:h2:" + createTempFile();
```

Comment:
Indeed. I will refactor the tests, so they are not tied to a file-based database.